master/pkg/model/task.go

```diff
@@ -208,6 +211,23 @@ func (s AllocationState) Proto() taskv1.State {
 	}
 }
 
+// Proto returns the proto representation of the allocation state.
+func (a Allocation) Proto() *taskv1.Allocation {
+	startTime := a.StartTime.String()
+	endTime := a.EndTime.String()
+	return &taskv1.Allocation{
+		TaskId:       a.TaskID.String(),
```

Comment:
super nit but on ```AllocationId``` we do ```string(a.AllocationID)``` while on TaskID we do ```a.TaskID.String()```

I personally prefer ```string(a.TaskID)``` but not by much, I guess ```string(``` feels more explicit since we are doing a type conversion. 

I guess its kinda weird we have a ```.String()``` method on these types in my opinion.

Reply:
made it string(a.TaskID)